cardano-cli/src/Cardano/CLI/Types/Errors/TxValidationError.hs

```diff
@@ -321,22 +325,50 @@ votingScriptWitnessSingleton votingProcedures (Just scriptWitness) =
   let voter = fromJust $ getVotingScriptCredentials votingProcedures
   in Map.singleton voter scriptWitness
 
+newtype TxGovDuplicateVotes era =
+  TxGovDuplicateVotes [L.GovActionId (L.EraCrypto (ShelleyLedgerEra era))]
```

Comment:
We're using `StandardCrypto` everywhere anyway. This would simplify the type a bit and you wouldn't have to use existential.
```suggestion
newtype TxGovDuplicateVotes =
  TxGovDuplicateVotes [L.GovActionId StandardCrypto)]
```